frontend/packages/helm-plugin/integration-tests/features/helm/helm-navigation.feature

```diff
@@ -0,0 +1,126 @@
+@helm
+Feature: Navigations on Helm Chart
+              As a user, I want to navigate to different pages related to Helm Charts
```

Comment:
Indentation

Reply:
Updated automatically on save and not able to see any gherkin-lint issue